src/components/Filters/components/FilterValue/FilterValue.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import {Text} from '@gravity-ui/uikit';
+import {block} from 'src/components/utils/cn';
+
+import {RemoveFilterButton} from '../RemoveFilterButton';
+
+import './FilterValue.scss';
+
+const b = block('filter-value');
+
+interface FilterValueProps {
+    className?: string;
+
+    onClick?: React.MouseEventHandler<HTMLButtonElement>;
+    onClear: VoidFunction;
+
+    fieldName: React.ReactNode;
+    fieldValue?: React.ReactNode;
+    emptyValueText?: React.ReactNode;
+
+    hideFieldName?: boolean;
+}
+
+export function FilterValue(props: FilterValueProps) {
+    const {onClick, onClear, fieldName, fieldValue, emptyValueText, hideFieldName, className} =
+        props;
+
+    const WrapperTag = typeof onClick === 'function' ? 'button' : 'span';
+
+    const renderDelimiter = fieldValue || emptyValueText;
+
+    return (
+        <div className={b(null, className)}>
+            <WrapperTag className={b('content', {type: WrapperTag})} onClick={onClick}>
```

Comment:
Why can't we just use Label component for this?

Reply:
We are discussing it [here](https://github.com/gravity-ui/components/pull/74#discussion_r1276835819)